src/binding.cc

```diff
@@ -3073,9 +3073,9 @@ int v8__ModuleRequest__GetSourceOffset(const v8::ModuleRequest& self) {
   return self.GetSourceOffset();
 }
 
-const v8::FixedArray* v8__ModuleRequest__GetImportAssertions(
+const v8::FixedArray* v8__ModuleRequest__GetImportAttributes(
     const v8::ModuleRequest& self) {
-  return local_to_ptr(self.GetImportAssertions());
```

Comment:
https://chromium-review.googlesource.com/c/v8/v8/+/5205273

Reply:
They finally renamed it